systemd/ceph-rgw.tmpfiles.d

```diff
@@ -0,0 +1,2 @@
+# create rgw socket directory
+d /var/run/ceph-rgw 0755 wwwrun www - -
```

Comment:
How can we conditionalize these `wwwrun` and `www` values? Maybe autoconf flags for `--with-radosgw-user` and `--with-radosgw-group` that we can set per-distro?

On RHEL we would set these to "root" due to http://tracker.ceph.com/issues/11453 , until http://tracker.ceph.com/issues/9133 gets resolved.

A couple other questions came to mind:

- Does SUSE have a `wwwrun` UID defined if the apache2 package is not installed on the system?
- How are you telling RGW to use this alternate socket directory?